WcaOnRails/app/webpacker/components/StaticPages/Disclaimer/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { List } from 'semantic-ui-react';
+
+function Disclaimer() {
+  return (
+    <List as="ul">
+      <List.Item as="li">
```

Comment:
What happens if you omit the `as=ul` and `as=li`?

Reply:
You lose the bullet points, I just followed the docs for created a bullet point list...